packages/web-components/fast-foundation/src/breadcrumb/breadcrumb.ts

```diff
@@ -0,0 +1,49 @@
+import { FASTElement, observable, attr } from "@microsoft/fast-element";
+import { BreadcrumbItem } from "../breadcrumb-item";
+
+/**
+ * A Breadcrumb Custom HTML Element.
+ *
+ * @public
+ */
+export class Breadcrumb extends FASTElement {
+    /**
+     * @internal
+     */
+    @observable
+    public slottedBreadcrumbItems: HTMLElement[];
+    public slottedBreadcrumbItemsChanged() {
+        if (this.$fastController.isConnected) {
+            const lastNode: HTMLElement = this.slottedBreadcrumbItems[
```

Comment:
can slottedBreadcrumItems be empty? might not want to do some of this code if so

Reply:
created a checked to see slottedBreadcrumbItems is undefined